addons/binding/org.openhab.binding.wifiled/src/main/java/org/openhab/binding/wifiled/handler/LEDStateDTO.java

```diff
@@ -149,9 +154,9 @@ public LEDStateDTO withIncrementedProgramSpeed(int step) {
     }
 
     public Color getColor() {
-        float hue = (float) this.getHue().intValue() / 360;
-        float saturation = (float) this.getSaturation().doubleValue() / 100;
-        float brightness = (float) this.getBrightness().intValue() / 100;
+        float hue = (float) this.getHue().doubleValue() / 360.0f;
```

Comment:
here is a though...

I can get rid of getColor()... and use bit shifting in sendLEDData

So
```
            byte r = (byte) (ledState.getColor().getRed() & 0xFF);
            byte g = (byte) (ledState.getColor().getGreen() & 0xFF);
            byte b = (byte) (ledState.getColor().getBlue() & 0xFF);
```

becomes:
```
            byte r = (byte) (ledState.getRGB()>>16 & 0xFF);
            byte g = (byte) (ledState.getRGB()>>8 & 0xFF);
            byte b = (byte) (ledState.getRGB() & 0xFF);
```

getRGB() is already part of HSBType anyways... There is no need to create a Color object and do the calculations for the solo purpose of getting RGB values while HSBType already can do that...